contracts/citycoin.clar

```diff
@@ -426,31 +463,31 @@
 ;; * This miner hasn't mined in this block before
 ;; * The miner is committing a positive number of uSTX
 ;; * The miner has the uSTX to commit
-(define-read-only (can-mine-tokens (miner-id principal)
-                                   (stacks-bh uint)
-                                   (amount-ustx uint)
-                                   (miners-rec { 
-                                       miners: (list 32 { miner: principal, amount-ustx: uint }),
-                                       claimed: bool
-                                   }))
-
-    (begin
-        (asserts! (is-some (get-reward-cycle stacks-bh))
-            (err ERR-STACKING-NOT-AVAILABLE))
+(define-read-only (can-mine-tokens (miner principal) (stacks-bh uint) (amount-ustx uint))
```

Comment:
Would it be more efficient to pass in both `miner principal` and `miner-id uint` here, rather than doing the extra lookup?

In `mine-tokens` the value is known by calling `get-or-create-miner-id`